playbooks/roles/ses-rook/tasks/main.yml

```diff
@@ -89,6 +89,37 @@
   retries: 30
   delay: 15
 
+# kubectl cp cannot be use due to missing tar in the container
+- name: Get mons from ceph.conf of the ceph tools container
+  shell: |
+    set -o pipefail
+    kubectl -n rook-ceph exec $(kubectl -n rook-ceph get pod -l 'app=rook-ceph-tools' -o jsonpath='{.items[0].metadata.name}') -- \
+    sed -n 's/mon_host = //p' /etc/ceph/ceph.conf
```

Comment:
yikes. Arent the mons stored somewhere else? In secrets or configmaps? Maybe we can get them from there?

Reply:
This is indeed just one option. 
Other one that I tested  is checking IP's of the mon pods like `"kubectl get pods -n rook-ceph -l app=rook-ceph-mon -o json | jq '.items[].status.podIP'"`

But getting them from ceph.conf seems to be similar to what we did in AIO case (not that it matters that much...)

Reply:
It's actually possible we do not need the mon values at all for OSH charts and rook backend should take care of everything(?)

Reply:
I can do e.g. this:


```kubectl get cm rook-ceph-mon-endpoints -n rook-ceph -o json | jq '.data["data"] | split(",")'
[
  "a=10.111.157.172:6789",
  "b=10.108.109.95:6789",
  "c=10.111.196.230:6789"
]
```

but that's not much better